main.js

```diff
@@ -934,9 +936,36 @@
             registerServerdiscovery();
             registerWakeOnLan();
 
-            // and load the index.html of the app.
-            mainWindow.loadURL(url);
-
+            console.log("Break Point 0");
+            stats = fs.lstatSync("database.txt");
+            var url = "";
+
+            try {
+                // Is it a file?
+                if (stats.isFile()) {
+                    console.log("File Validated, connection.txt is there");
+                    fs.readFile("database.txt", 'utf8', function(err, contents) {
+                        url = contents;
+                        console.log("The contents of database.txt is [" + url + "]");
+                        // and load the index.html of the app.
+                        console.log("Break Point 1");
+                        mainWindow.loadURL(url);
```

Comment:
Does doing this still support the correct rendering/layout of the old electron app?